### PR TITLE
New translations for translation/ru/sv/sutta/sn/sn1

### PR DESCRIPTION
**Pull request by sc-translatatron**
Request made by neptunao

Please do not modify this branch directly. Changes should be
made via the Bilara Translation App and the Pull Request
updated from Bilara.

### CODE REVIEW

**Comment by ccronje** [comment]
Note to self: newline characters SN 1.20

**Comment by neptunao** [comment]
> Note to self: newline characters SN 1.20

I checked in vscode and could not find any. Could you please give me a line number so I fix it?

File: translation/ru/sv/sutta/sn/sn1/sn1.20_translation-ru-sv.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  "sn1.20:5.1": "«Я не оставил того, что видно здесь и сейчас, подруга, ради преследования того, что требует времени. ",
+  "sn1.20:5.2": "Я оставил то, что требует времени, ради преследования того, что видно здесь и сейчас. ",
+  "sn1.20:5.3": "Ведь Благословенный, подруга, утверждал, что чувственные удовольствия съедают много времени, полны страданий, полны отчаяния, а опасность, заключённая в них, ещё больше. ",
+  "sn1.20:5.4": "А эта Дхамма видима здесь и сейчас, незамедлительно действенная, приглашающая к исследованию, ведущая вперёд, переживаемая мудрыми для себя». ",
+  "sn1.20:6.1": "«Но как так, монах, что Благословенный утверждал, что чувственные удовольствия съедают много времени, полны страданий, полны отчаяния, а опасность, заключённая в них, ещё больше? ",
+  "sn1.20:6.2": "А эта Дхамма видима здесь и сейчас, незамедлительно действенная, приглашающаяк исследованию, ведущая вперёд, переживаемая мудрыми для себя?». ",
+  "sn1.20:7.1": "«Я \n                  недавно получил монашеское посвящение, подруга, недавно оставил \n                  жизнь домохозяйскую ради жизни бездомной, недавно пришёл в эту \n                  Дхамму и Винаю. ",
```
Comment (ccronje) [inline translation/ru/sv/sutta/sn/sn1/sn1.20_translation-ru-sv.json]: @neptunao from this line onwards there are 29 either newline chars or double straight quotation marks - thanks

**Comment by neptunao** [comment]
@ccronje I will recreate this PR with fixed commits.

**Pull request closed without merging**